build.cmd

```diff
@@ -10,25 +13,34 @@ IF "%*"=="" (
   SET SCRIPTARGS=%*
 )
 
-IF EXIST %CACHED_NUGET% goto copynuget 
-echo Downloading latest version of NuGet.exe... 
-IF NOT EXIST %LocalAppData%\NuGet md %LocalAppData%\NuGet 
-@powershell -NoProfile -ExecutionPolicy unrestricted -Command "$ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest 'https://www.nuget.org/nuget.exe' -OutFile '%CACHED_NUGET%'" 
- 
-:copynuget 
-IF EXIST .nuget\nuget.exe goto restore 
-md .nuget 
-copy %CACHED_NUGET% .nuget\nuget.exe > nul 
- 
-:restore 
-IF EXIST packages\KoreBuild goto run  
-.nuget\NuGet.exe install KoreBuild -ExcludeVersion -o packages -nocache -pre -Source https://myget.org/F/aspnetvnext
```

Comment:
For the most part the template build pulls in packages from the internal machines and use those as source. I'd assume most people do have good feeds. If it trips people up we can probably look at adding a config file